tw_pywrap/tower.py

```diff
@@ -55,11 +56,21 @@ def _tw_run(self, cmd, *args, **kwargs):
         logging.debug(f" Running command: {full_cmd}\n")
 
         # Run the command and return the stdout
-        process = subprocess.Popen(full_cmd, stdout=subprocess.PIPE, shell=True)
+        process = subprocess.Popen(
+            full_cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, shell=True
+        )
         stdout, _ = process.communicate()
         stdout = stdout.decode("utf-8").strip()
 
-        return stdout
+        # Check if an error occurred
+        # ruff: noqa: E501
+        if stdout:
+            if re.search(r"ERROR: .* already exists", stdout):
```

Comment:
Maybe I'm missing how things work fundamentally in these scripts but I was hoping we could catch this **before** the ERROR occurs and not afterwards.

Reply:
Lets maybe have a chat later.